src/PostgREST/Config/Database.hs

```diff
@@ -203,3 +204,14 @@ param = HE.param . HE.nonNullable
 
 arrayParam :: HE.Value a -> HE.Params [a]
 arrayParam = param . HE.foldableArray . HE.nonNullable
+
+-- |
+-- >>> quotePreConfFunc "postgrest.pre_config"
+-- "\"postgrest\".\"pre_config\""
+--
+-- >>> quotePreConfFunc "true"
+-- "\"true\""
+quotePreConfFunc :: Text -> Text
+quotePreConfFunc f = T.intercalate "." $ map escapeIdent $ T.split (== '.') f
+    where
+      escapeIdent id = "\"" <> id <> "\""
```

Comment:
Don't we have a helper function for this somewhere already?

Reply:
> I think long term solution would be to have our own Prelude as discussed in https://github.com/PostgREST/postgrest/issues/1804#issuecomment-3254810274. For now I have handled it by quoting it by a new function quoteQi.

Maybe instead we can move 

https://github.com/PostgREST/postgrest/blob/4643eaf7bf9a3633720e619e7eea0ad3b705034a/src/PostgREST/Query/SqlFragment.hs#L166-L167

To `src/PostgREST/SchemaCache/Identifiers.hs`?

Reply:
> Maybe instead we can move
> [...] 
> To `src/PostgREST/SchemaCache/Identifiers.hs`?

Yes, lemme refactor this. :+1: